VikingQuiz.Api/Controllers/QuizzesController.cs

```diff
@@ -70,36 +75,41 @@ public class QuizzesController : Controller
         [HttpPost]
         [RequestSizeLimit(5_000_000)]
         [Authorize]
-        public async Task<IActionResult> AddQuiz(NewQuizViewModel quizBodyData) {
+        public async Task<IActionResult> AddQuiz(NewQuizViewModel quizBodyData)
+        {
             int currentUserId = User.Claims.GetUserId();
             IActionResult imageHttpResponse = FileValidityChecker(quizBodyData);
-            if (IActionResult.Equals(imageHttpResponse, Ok())) {
+            if (IActionResult.Equals(imageHttpResponse, Ok()))
+            {
                 return imageHttpResponse;
             }
 
             string fileUrl;
             AzureBlobService blobService;
-            try {
-                blobService = new AzureBlobService();
-                await blobService.InitializeBlob();
-                fileUrl = await blobService.UploadPhoto(quizBodyData.Files[0]);
+            try
+            {
+                blobService = new AzureBlobService(azureContainerName);
+                fileUrl = await blobService.UploadPhotoAsync(quizBodyData.Files[0]);
             }
-            catch (Exception) {
+            catch (Exception)
+            {
                 return BadRequest("Image could not be uploaded.");
             }
 
 
-            Quiz createdQuiz = quizRepository.CreateQuiz(new Quiz {
+            Quiz createdQuiz = quizRepository.CreateQuiz(new Quiz
+            {
                 Title = quizBodyData.Title,
                 PictureUrl = fileUrl,
                 UserId = currentUserId
             });
```

Comment:
The table Quiz has a column named LastModified which is null by default. Maybe when you create a new quiz you should set the value to current date and time.